src/Nri/Ui/Table/V5.elm

```diff
@@ -0,0 +1,285 @@
+module Nri.Ui.Table.V5 exposing
+    ( Column, custom, string
+    , view, viewWithoutHeader
+    , viewLoading, viewLoadingWithoutHeader
+    )
+
+{-| Upgrading from V1:
```

Comment:
Should this have instructions for upgrading from V4?

Reply:
Nice catch. Added